linkcheckbroken is broken

### Plone Documentation versions affected

- [ ] 5.x
- [X] 6.x

### Description

`make linkcheckbroken` returns an unhelpful error message:

`make: *** [linkcheckbroken] Error 1`

However:

`make linkcheck | GREP_COLORS='0;31' grep -wi "broken\|redirect" --color=auto`

Works just fine. This is identical to the former in `Makefile`.

The only thing that might be causing the error is possibly the check for `html_meta` values returning a non-zero exit code.

This issue affects `plone/volto` and `plone/plone.restapi` as well, except for prefixing the commands with `docs-`.

For now we have changed the CI to use `make linkcheck` instead of `make linkcheckbroken` across all the repos.

@wolbernd sorry, I should have put this into proper context.

## documentation

```
$  make linkcheckbroken
cd ./docs/ && /Users/stevepiercy/projects/Plone/documentation/bin/sphinx-build -b linkcheck -d ../_build//doctrees   . ../_build//linkcheck | GREP_COLORS='0;31' grep -wi "broken\|redirect" --color=auto
make: *** [linkcheckbroken] Error 1
```

## documentation/submodules/plone.restapi

```
$ cd submodules/plone.restapi
$  make docs-linkcheckbroken
cd ./docs/source/ && /Users/stevepiercy/projects/Plone/documentation/submodules/plone.restapi/bin/sphinx-build -b linkcheck -d ../_build//doctrees  . ../_build//linkcheck | GREP_COLORS='0;31' egrep -wi broken --color=auto
make: *** [docs-linkcheckbroken] Error 1
```

## documentation/submodules/volto

```
$ cd submodules/volto
$  make docs-linkcheckbroken
make: *** [docs-linkcheckbroken] Error 1
```

It's also a problem in CI/CD.

Linkcheck does not check for meta tag warnings.

--- a/Makefile
+++ b/Makefile
@@ -212,7 +212,7 @@ doctest: deps
 	      "results in $(BUILDDIR)/doctest/output.txt."
 
 .PHONY: test
-test: clean linkcheckbroken spellcheck  ## Clean docs build, then run linkcheck, spellcheck
+test: clean linkcheckbroken spellcheck  ## Clean docs build, then run linkcheckbroken, spellcheck
 
 .PHONY: deploy
 deploy: clean html

--- a/Makefile
+++ b/Makefile
@@ -212,7 +212,7 @@ doctest: deps
 	      "results in $(BUILDDIR)/doctest/output.txt."
 
 .PHONY: test
-test: clean linkcheck spellcheck  ## Clean docs build, then run linkcheck, spellcheck
+test: clean linkcheckbroken spellcheck  ## Clean docs build, then run linkcheck, spellcheck
 
 .PHONY: deploy
 deploy: clean html

--- a/Makefile
+++ b/Makefile
@@ -189,7 +189,7 @@ linkcheck: deps  ## Run linkcheck
 
 .PHONY: linkcheckbroken
 linkcheckbroken: deps  ## Run linkcheck and show only broken links
-	cd $(DOCS_DIR) && $(SPHINXBUILD) -b linkcheck $(ALLSPHINXOPTS) $(BUILDDIR)/linkcheck | GREP_COLORS='0;31' grep -wi "broken\|redirect" --color=auto
+	cd $(DOCS_DIR) && $(SPHINXBUILD) -b linkcheck $(ALLSPHINXOPTS) $(BUILDDIR)/linkcheck | GREP_COLORS='0;31' grep -wi "broken\|redirect" --color=auto || test $$? = 1
 	@echo
 	@echo "Link check complete; look for any errors in the above output " \
 		"or in $(BUILDDIR)/linkcheck/ ."

